scripts/create_version.sh

```diff
@@ -139,6 +139,7 @@ create_branch_for_new_release() {
     " data/args.yml
 
     UPDATE_BRANCH=${NEW_RELEASE_BRANCH} make update-common
+    make gen
```

Comment:
This would be running on the release-1.9 branch. We need to add this to "advance_master_to_next_release" section

Reply:
Updated.